bin/cron/reschedule.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const Cli = require('structured-cli');
+const Cron = require('../../lib/cron');
+const PrintCronJob = require('../../lib/printCronJob');
+
+module.exports = Cli.createCommand('reschedule', {
+    description: 'Change the timing of a cron job',
+    plugins: [require('../_plugins/profile')],
+    optionGroups: {
+        'Cron options': {
+            schedule: {
+                description:
+                    'Either a cron-formatted schedule (see: http://crontab.guru/) or an interval of hours ("h") or minutes ("m"). Note that not all intervals are possible.',
+                type: 'string',
+            },
+            state: {
+                description: "Set the cron job's state",
+                choices: ['active', 'inactive'],
+                type: 'string',
+            },
+            tz: {
+                description: `An IANA timezone name (see: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), relative to which the cron schedule will be calculated. If not specified, the webtask cluster's default timezone is used.`,
+                metavar: 'TIMEZONE',
+                type: 'string',
+            },
+        },
+    },
+    params: {
+        name: {
+            description: 'The name of the cron job to update.',
```

Comment:
It's maybe a bit confusing to use the term 'update' here when there is also an `wt cron update` cmd

Reply:
Fixed in 6a3a0ac.